PowerUp/app/src/main/res/layout/activity_vocab_match_game.xml

```diff
@@ -4,80 +4,85 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="powerup.systers.com.vocab_match_game.VocabMatchGameActivity"
-    android:background="@drawable/vocab_bg">
+    android:background="@drawable/vocab_bg"
+    tools:context="powerup.systers.com.vocab_match_game.VocabMatchGameActivity">
+
     <TextView
         android:id="@+id/vocab_score"
         android:layout_width="80dp"
         android:layout_height="80dp"
         android:background="@drawable/vocab_score"
-        android:text="0"
         android:gravity="center"
+        android:text="@string/zero_text"
         android:textSize="25sp"
-        android:textStyle="bold"/>
+        android:textStyle="bold" />
+
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:baselineAligned="false"
```

Comment:
This does not concerns with removing hard coded text. Revert this.

Reply:
What name should I give for this commit? @Rimjhim28

Reply:
@pprathameshmore Depends on your perception. Maintain 1 commit per PR.

Reply:
@Rimjhim28 I made changes as per request! Please review! 👍